### ISSUE TITLE
[HIGH] Triple VDM LR schedulers are created but never stepped

### ISSUE THREAD

**Issue by Maxelee** [Maxelee/vdm_BIND#5]
## Description

In `vdm/vdm_model_triple.py`, the `configure_optimizers()` method creates three separate CosineAnnealingLR schedulers (one per channel VDM), but `training_step()` never calls `scheduler.step()`, and the return format does not use Lightning's scheduler protocol correctly.

## Current Code (lines ~447-450)

```python
schedulers = [
    torch.optim.lr_scheduler.CosineAnnealingLR(opt, T_max=self.hparams.n_steps)
    for opt in optimizers
]
return optimizers, schedulers
```

The schedulers are returned but Lightning requires a specific dict format for multiple optimizers/schedulers. The current tuple return is only valid for single optimizer setups.

## Impact

- Learning rate never decays during Triple VDM training
- Training uses constant LR throughout, which may explain suboptimal convergence
- The schedulers are silently ignored

## Fix

Return schedulers in Lightning's expected format:

```python
return (
    [{"optimizer": opt, "lr_scheduler": {"scheduler": sched, "interval": "step"}}
     for opt, sched in zip(optimizers, schedulers)]
)
```

Or manually step schedulers in `training_step()`.

## Files
`vdm/vdm_model_triple.py` lines 440-455

**Comment by Maxelee** [comment]
Fixed in commit 6f910ac on branch fix/critical-and-high-issues
